### PR TITLE
rules #48

### PR DESCRIPTION
**Pull request by sourTaste000**
#48

### CODE REVIEW

**Comment by sourTaste000** [comment]
super retarded code

File: rules.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "5": "No advertising",
+  "6": {
+    "a": "No spamming",
+    "b": "This includes repeatedly pinging or dming someone",
+    "c": "This includes pinging multiple high ranking members for no reason at all"
+  },
+  "7": "No using an alt to avoid mutes or bans, it counts as ban evading",
+  "8": "No trolling :wheelchair:",
+  "9": "Follow discord TOS: https://discordapp.com/terms",
+  "10": "Follow discord guidelines: https://discordapp.com/guidelines",
```
Comment (dominikaaaa) [inline rules.json:29]: Could these please have <> around the links, so they don't embed? eg <https://google.com>

File: rules.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "d": "No promoting \"cracked\" servers",
+    "e": "No promoting hacking or cheating on servers which don't allow it",
+    "f": "No discussing any of the above"
+  },
+  "4": "No NSFW period",
+  "5": "No advertising",
+  "6": {
+    "a": "No spamming",
+    "b": "This includes repeatedly pinging or dming someone",
+    "c": "This includes pinging multiple high ranking members for no reason at all"
```
Comment (dominikaaaa) [inline rules.json:24]: This is in _italics_ on discord

File: commands/rules.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            message.channel.send(rules["9"]);
+            break;
+        case "10":
+            message.channel.send(rules["10"]);
+            break;
+        case "secret":
+            message.channel.send(rules["message-from-sasha"]);
+            break;
+        default:
+            message.channel.send("Provide an input dumbass")
```
Comment (dominikaaaa) [inline commands/rules.js:79]: Maybe change this to "Please provide a rule number"?

**Pull request merged**